lib/solr_wrapper/instance.rb

```diff
@@ -103,35 +103,31 @@ def restart
     ##
     # Check the status of a managed Solr service
     def status
```

Comment:
This seems like a backwards-compatible change in behavior for `#status`? Could we extract this behavior to a different method name instead?